desafio13.sql

```diff
@@ -0,0 +1,6 @@
+SELECT p.ProductName AS Produto,
+  p.Price AS Preço
+FROM w3schools.products AS p
+  INNER JOIN w3schools.order_details AS od ON p.ProductID = od.ProductID
+WHERE od.Quantity > 80
```

Comment:
Ribas aprendi com o @yoshidalucas92 que quando vc ta usando ON vc não precisa usar o where uma AND ali no ON ja funciona e o codigo fica um pouco mais bonitinho, ai to passando esse aprendizado pra frente.
```suggestion
  INNER JOIN w3schools.order_details AS od ON p.ProductID = od.ProductID AND od.Quantity > 80
```

Reply:
Boooa @Ronan-Fernandes !!
@ribaslucas ,
Só para complementar o que o Ronan sugeriu,
Existe uma recomendação de usar `WHERE` somente se for realmente necessário.
No caso de JOINS é possível adicionar outra condição.
Usando sua resolução como exemplo, ficaria mais ou menos assim:
  ```
INNER JOIN w3schools.products AS p
  ON od.ProductID = p.ProductID AND od.Quantity > 80
```
obs: Se usar somente `JOIN`, já fica sub entendido que se trata de um `INNER JOIN`, pois é o único que não precisa ser especificado, ao contrario dos `LEFT` e `RIGHT JOIN`